app/services/food_service.rb

```diff
@@ -0,0 +1,20 @@
+class FoodService
+  class << self
+    def foods_containing(keyword)
+      response = conn.get("fdc/v1/foods/search?query=#{keyword}")
+      parse_data(response)
+    end
```

Comment:
you could utilize an optional parameter from the api, `per_page` to get exactly 10 results back

Reply:
Yea! I will look into pagination! I did considered putting in the facade.  i know have it locally as: 

  def foods_containing(keyword)
      foods = FoodService.foods_containing(keyword)
      # binding.pry
      foods[:foods][0..9].each do |food_info|
        Food.new(food_info)
      end
    end